apps/etterlatte-behandling/src/main/kotlin/oppgave/kommentar/OppgaveKommentarService.kt

```diff
@@ -0,0 +1,20 @@
+package no.nav.etterlatte.oppgave.kommentar
+
+import no.nav.etterlatte.libs.common.oppgave.OppgaveKommentarDto
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import java.util.UUID
+
+class OppgaveKommentarService(
+    private val oppgaveKommentarDao: OppgaveKommentarDao,
+) {
+    private val logger: Logger = LoggerFactory.getLogger(this.javaClass.name)
+
+    fun hentOppgaveKommentarer(oppgaveId: UUID): List<OppgaveKommentarDto> {
+        return oppgaveKommentarDao.hentKommentarer(oppgaveId) // TODO sortert?
+    }
+
+    fun opprettKommentar(oppgaveKommentarDto: OppgaveKommentarDto) {
+        oppgaveKommentarDao.opprettKommentar(oppgaveKommentarDto)
+    }
```

Comment:
Backend bør ikke bare stole på det som kommer fra frontend her, men opprette en kommentar i saken til mottatt oppgave-id, med opprettet tidspunkt nå, og med identen til saksbehandleren som oppretter oppgaven -- for å sikre at man ikke kan putte inn det man vil her